src/app/preferences/_preferences.html

```diff
@@ -8,7 +8,7 @@
 		{{'PREFERENCES_MANAGE_SETTINGS_PREFERENCES' | translate}}<br />
 	</p>
 	
-    <div class="menu">
+    <!--<div class="menu">
```

Comment:
No, you do not have to duplicate the HTML page.
Just keep **_preferences-language.html** and update the module's **routes.js** file.